Descriptions are now displayed on list of /membership /groupings /admin

JorWo opened the pull request:
# Ticket Link

[Groupings-533](https://www.hawaii.edu/jira/browse/GROUPINGS-533)

# List of squashed commits

- Added margin to description icon

# Test Checklist

- [x] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [x] General Visual Inspection:

mhodgesatuh reviewed (commented):
let endpoint = BASE_URL + ""
This look superfluous.  What's the point?

JorWo commented:
> let endpoint = BASE_URL + "" This look superfluous. What's the point?

I believe I removed it to test the get request and I just added "/" back in.

devgav reviewed (approved):
Let's wait for the API to be approved as well before I commit this. Seems like these UI changes rely on the data from the API.
